sdk/contentsafety/Azure.AI.ContentSafety/README.md

```diff
@@ -56,32 +104,307 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 ## Examples
 
-You can familiarize yourself with different APIs using [Samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/contentsafety/Azure.AI.ContentSafety/tests/Samples).
+The following section provides several code snippets covering some of the most common Content Safety service tasks, including:
+
+- [Analyze text](#analyze-text)
+- [Analyze image](#analyze-image)
+- [Manage text blocklist](#manage-text-blocklist)
+
+Please refer to [sample data](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/contentsafety/Azure.AI.ContentSafety/tests/Samples/sample_data) for the data used here. For more samples, please refer to [Samples](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/contentsafety/Azure.AI.ContentSafety/tests/Samples).
+
+### Analyze text
+
+#### Analyze text without blocklists
+
+```C# Snippet:Azure_AI_ContentSafety_AnalyzeText
+string datapath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "Samples", "sample_data", "text.txt");
+string text = File.ReadAllText(datapath);
+
+var request = new AnalyzeTextOptions(text);
+
+Response<AnalyzeTextResult> response;
+try
+{
+    response = client.AnalyzeText(request);
+}
+catch (RequestFailedException ex)
+{
+    Console.WriteLine(String.Format("Analyze text failed.\nStatus code: {0}, Error code: {1}, Error message: {2}", ex.Status, ex.ErrorCode, ex.Message));
+    throw;
+}
+
+if (response.Value.HateResult != null)
+{
+    Console.WriteLine(String.Format("Hate severity: {0}", response.Value.HateResult.Severity));
+}
+if (response.Value.SelfHarmResult != null)
+{
+    Console.WriteLine(String.Format("SelfHarm severity: {0}", response.Value.SelfHarmResult.Severity));
+}
+if (response.Value.SexualResult != null)
+{
+    Console.WriteLine(String.Format("Sexual severity: {0}", response.Value.SexualResult.Severity));
+}
+if (response.Value.ViolenceResult != null)
+{
+    Console.WriteLine(String.Format("Violence severity: {0}", response.Value.ViolenceResult.Severity));
+}
```

Comment:
This is awfully verbose. I suggest the null coalescence operator:
```suggestion
Console.WriteLine("Hate severity: {0}", response.Value.HateResult?.Severity ?? "0.0");
Console.WriteLine("SelfHarm severity: {0}", response.Value.SelfHarmResult?.Severity ?? "0.0");
Console.WriteLine("Sexual severity: {0}", response.Value.SexualResult?.Severity ?? "0.0");
Console.WriteLine("Violence severity: {0}", response.Value.ViolenceResult?.Severity ?? "0.0");
```
`Console.WriteLine` also implements string formatting more efficiently, so you're wasting time and showing atypical code so don't use `string.format`.

Reply:
Fixed.